requirements.txt

```diff
@@ -1,3 +1,4 @@
-#PyMySQL==0.7.4
-#psycopg2==2.6.1
-robotframework>=3.0
+#PyMySQL==1.0.2
+#psycopg2==2.9.1
+robotframework>=4.0.0
```

Comment:
Unless we require something from the new version of the RF, it's better to stick to lower version so more people can use library (for those who can't use RF 4.0 for some reason yet). WDYT @amochin ?

Reply:
Well, this requirements file doesn't define any dependencies for library users.
It's relevant for developers only - to extent the library code and run the tests.
Or so I was using this file before.

It's the pyproject.toml file, where are dependencies specified - for installing and using the library.
Currently it doesn't limit the RF version at all.

Regarding the library development, the **requirements.txt** - the current version in the master doesn't limit the RF version as well. However, the current tests use some things from later RF versions, so actually it makes sense to put an even higher limitations - like RF 5.0 maybe, would need to check it...